.ci/oci-dashboard-happy-path.sh

```diff
@@ -9,33 +9,195 @@
 #
 
 set -e
+set -o pipefail
+set -u
 set -x
 
-export CHE_REPO_BRANCH="main"
-export DEVWORKSPACE_HAPPY_PATH="https://raw.githubusercontent.com/eclipse/che/${CHE_REPO_BRANCH}/tests/devworkspace-happy-path"
-source <(curl -s ${DEVWORKSPACE_HAPPY_PATH}/common.sh)
+START=$(date +%s.%N)
+SCRIPT_DIR=$(dirname $(readlink -f "$0"))
+
+export WORKDIR=${WORKDIR:-"${SCRIPT_DIR}/workdir"}
+export CHE_NAMESPACE=${CHE_NAMESPACE:-"eclipse-che"}
+export HAPPY_PATH_POD_NAME='happy-path-che'
```

Comment:
cc: @dmytro-ndp is it ok to use happy-path naming or we should stick to smoke test 
just want to avoid confusion in the scripts

Reply:
good catch, @ibuziuk 

+1 to use `smoke-test` naming, because it would help to avoid wrong expectations from test result

Reply:
I replaced it at as many places as I was able to confidently do.